bpf/lib/process.h

```diff
@@ -535,4 +535,39 @@ execve_joined_info_map_get(__u64 tid)
 
 _Static_assert(sizeof(struct execve_map_value) % 8 == 0,
 	       "struct execve_map_value should have size multiple of 8 bytes");
+
+#define METRIC_EXEC	  0
+#define METRIC_CLONE	  1
+#define METRIC_EXIT	  2
+#define METRIC_DATA	  3
+#define METRIC_CGROUP	  4
+#define METRIC_LOADER	  5
+#define METRIC_TRACEPOINT 6
+#define METRIC_KPROBE	  7
+#define METRIC_UPROBE	  8
+#define METRIC_MAX_VALUES 9
+
```

Comment:
just a thought.. in my initial attempt I used MSG_OP_* values to identify metrics,
I wonder it's better than add new cathegories? also  might be  easier then in
output when comparing to tetragon_msg_op_total values? I assume MSG_OP_*
link to sensors directly anyway

then we might need to use BPF_MAP_TYPE_PERCPU_HASH because the calues are
scattered

Reply:
fwiw, I think using MSG_OP types makes sense.

> then we might need to use BPF_MAP_TYPE_PERCPU_HASH because the values are
scattered

They are not that scattered so I think it's fine to use an array to avoid the hash lookup.

Reply:
Make sense to me as well. The values that we care are:
```
  MSG_OP_EXECVE = 5,
  MSG_OP_EXIT = 7,
  MSG_OP_GENERIC_KPROBE = 13,
  MSG_OP_GENERIC_TRACEPOINT = 14,
  MSG_OP_GENERIC_UPROBE = 15,
  MSG_OP_CLONE = 23,
  MSG_OP_DATA = 24,
  MSG_OP_CGROUP = 25,
```
So an array of 26 elements seems to be enough.

Reply:
Let's go crazy and add 256.

Reply:
Updated the PR based on that comment.